src/tactic_script.fun

```diff
@@ -59,11 +53,11 @@ struct
 
   and parseRepeat D () =
         middle (symbol "*{") ($ (parseScript D)) (symbol "}")
-        wth LIMIT
+        wth REPEAT
```

Comment:
Is there a reason to change this to `REPEAT` other than the other identifier names? If not, feel free to change it all to limit...